ui/webui/resources/br_elements/br_toolbar/br_toolbar.ts

```diff
@@ -105,17 +105,26 @@ Polymer({
 
   /** @private */
   onMenuTap_: function() {
-    console.debug('[br_toolbar] Not Implemented: onMenuTap_')
+    this.dispatchEvent(new CustomEvent(
+      'cr-toolbar-menu-tap', {bubbles: true, composed: true}))
   },
 
   focusMenuButton() {
```

Comment:
I don't know if these (other than `onMenuTap_`) are needed but I thought I might as well implement them while I'm adding the button.